### PR TITLE
Include queue label in watchdog timer callback

### PR DESCRIPTION
**Pull request by adamjernst**

### CODE REVIEW

File: ComponentKit/Utilities/CKWatchdogTimer.mm [hunk 1]
```diff
@@ -18,9 +18,15 @@
 /** Protects timeoutNanoseconds and handler */
 static CK::StaticMutex mutex = CK_MUTEX_INITIALIZER;
 static uint64_t timeoutNanoseconds = 0;
-static void (*handler)(void) = nullptr;
+static void (*handler)(NSString *) = nullptr;
 
-static dispatch_source_t newTimer()
+static NSString *currentQueueLabel(void)
+{
+  const char *label = dispatch_queue_get_label(DISPATCH_CURRENT_QUEUE_LABEL);
```
Comment (eczarny) [inline ComponentKit/Utilities/CKWatchdogTimer.mm]: Would this always be the preparation queue's label?

Comment (adamjernst) [inline ComponentKit/Utilities/CKWatchdogTimer.mm]: Yes, so I was hoping we could customize the label for each?

Comment (eczarny) [inline ComponentKit/Utilities/CKWatchdogTimer.mm]: :+1:

**Comment by eczarny** [comment]
LGTM

**Pull request merged**